rts/Game/Camera/SpringController.cpp

```diff
@@ -186,14 +186,14 @@ float CSpringController::ZoomOut(const float3& curCamPos, const float2& zoomPara
 	if (!cursorZoomOut)
 		return 0.25f;
 
-	const float zoomInDist = CGround::LineGroundCol(curCamPos, curCamPos + mouse->dir * 150000.0f, false);
+	const float zoomInDist = CGround::LineGroundCol(curCamPos, curCamPos + newDir * 150000.0f, false);
```

Comment:
I'm pretty sure `newDir` should be irrelevant in this function. Zoom out for this camera always originates from center of screen.

However, I don't know how to adjust for this.